google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleStage.java

```diff
@@ -120,9 +119,20 @@ public void stage(@NotNull Path stagingDirectory) {
       if (isCustomRuntime) {
         FileUtils.copyDirectory(Paths.get(dockerDirectory).toFile(), stagingDirectory.toFile());
       }
-    } catch (IOException | InvalidPathException | MalformedYamlFileException ex) {
-      loggingHandler.print(ex.getMessage() + "\n");
-      throw new RuntimeException(ex);
+    } catch (Exception e) {
```

Comment:
stupid style thing: at one point we had a static analysis tool failing the build for one letter variables (including exception variables). I thought this was coming from the google guide, but I could be wrong. I personally didn't like that rule anyway.